### ISSUE TITLE
Friend Create: Allow compiling C / C++, etc. to WebAssembly

### ISSUE THREAD

**Issue by andresclari** [FriendUPCloud/friendup#76]
Would be really great if we could use Friend Create to compile C / C++ code, etc. to WebAssembly to allow for more complex apps and ports of native components.

This could be done by implementing a way for Friend Create, to send the file back to the backend server, run it through llvm -> wasm and retrieving the result and stdout output.

Then you could just link it to the application and execute it.

At some point this probably will be a requirement for complex applications to be written and ported from within Friend, imagine audio / video / image processing applications.

**Comment by stefkosdev** [comment]
Ok, thank you for your input. We will talk about this here and come back to you.

**Comment by titlestad** [comment]
Yes, it is planned to incorporate WASM in Friend Create - with the possibility to "compile" Python etc so you can use different languages to write Friend applications. Still a bit early days, but feel free to join the open source project and help out. Right now, Friend Create is mostly my babe, but I'd love to have help. Friend Create is a great opportunity to lift web development to the next level.
